vm-migration/src/lib.rs

```diff
@@ -59,6 +59,12 @@ pub enum MigratableError {
 
     #[error("Failed to complete migration for migratable component: {0}")]
     CompleteMigration(#[source] anyhow::Error),
+
+    #[error("Failed to coredump coredumpable component: {0}")]
+    Coredump(#[source] anyhow::Error),
+
+    #[error("File error: {0}")]
+    CoredumpFile(#[source] std::io::Error),
```

Comment:
I think you can remove these since you added them as part of `GuestDebuggableError`.

Reply:
Right.